extensions/wikia/FirstContributions/FirstContributions.setup.php

```diff
@@ -0,0 +1,17 @@
+<?php
+$wgExtensionCredits['api'][] = [
+	'name' => 'First Contributions',
+	'description' => 'Implements API module to list first edits of new editors on a wiki',
+	'descriptionmsg' => 'firstcontributions-desc',
+	'version' => 1,
+	'author' => 'Wikia',
+	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/FirstContributions',
+];
+
+$wgExtensionMessagesFiles['FirstContributions'] = __DIR__ . '/FirstContributions.i18n.php';
+
+$wgAutoloadClasses['ApiQueryFirstContributions'] = __DIR__ . '/api/ApiQueryFirstContributions.php';
+
+$wgAPIListModules['firstcontributions'] = 'ApiQueryFirstContributions';
+
+return true; // ...to the single purpose of the moment.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)